pep-0649.rst

```diff
@@ -1,13 +1,14 @@
 PEP: 649
 Title: Deferred Evaluation Of Annotations Using Descriptors
 Author: Larry Hastings <larry@hastings.org>
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Topic: Typing
 Content-Type: text/x-rst
 Created: 11-Jan-2021
-Post-History: 11-Jan-2021, 11-Apr-2021, 19-Apr-2023
-
+Python-Version: 3.13
+Post-History: 11-Jan-2021, 11-Apr-2021, 19-Apr-2023, 10-May-2023
```

Comment:
```suggestion
Post-History: `11-Jan-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/5QMMCRF4HTRRNJV56CGHVI5GRHVBDGQO/>`__,
              `12-Apr-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/QSASX6PZ3LIIFIANHQQFS752BJYFUFPY/>`__,
              `18-Apr-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/WUZGTGE43T7XV3EUGT6AN2N52OD3U7AE/>`__,
              `09-Aug-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/2MEOWHCVDLPABOBLYUGRXVOOOBYOLLU6/>`__,
              `20-Oct-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/SZLWVYV2HPLU6AH7DOUD7DWFUGBJGQAY/>`__,
              `20-Oct-2021 <https://discuss.python.org/t/type-annotations-pep-649-and-pep-563/11363>`__,
              `17-Nov-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/VIZEBX5EYMSYIJNDBF6DMUMZOCWHARSO/>`__,
              `15-Mar-2022 <https://discuss.python.org/t/finding-edge-cases-for-peps-484-563-and-649-type-annotations/14314>`__,
              `23-Nov-2022 <https://discuss.python.org/t/pep-649-deferred-evaluation-of-annotations-tentatively-accepted/21331>`__,
              `07-Feb-2023 <https://discuss.python.org/t/two-polls-on-how-to-revise-pep-649/23628>`__,
              `11-April-2023 <https://discuss.python.org/t/a-massive-pep-649-update-with-some-major-course-corrections/25672>`__,
```

After a bunch of post archeology on Discourse and Python-Dev, I've managed to, I believe, fairly accurately reconstruct the (looooooong, on this one) history of discussion threads for this PEP.

I've only listed threads that had as its core topic the discussion of PEP 649 (including comparison with the alternative PEP 563, but with 649 as a main focus), and limited it to those that were posted by either @larryhastings or the SC to gather PEP feedback (with the exception of [one thread](https://mail.python.org/archives/list/python-dev@python.org/thread/2MEOWHCVDLPABOBLYUGRXVOOOBYOLLU6/) by Eric V. Smith that was a continuation of Larry's previous PEP discussion thread, posted in apparent coordination with the same and was indeed the de-facto next iteration of the PEP discussion thread).

This included the threads that appeared to corresponding to the existing bare dates already listed (it seems there isn't a new one for this PEP update, so that can be added later if a new one is posted rather than posting to one of the already-listed threads).

@larryhastings , if you think something shouldn't be included or have something else that you think should be, just let me know of course!

Reply:
Goodness me!  You're doing yeoman's work here.  Thanks, CAM!

I'm catching a plane tonight and will have a busy day or two.  I'll try to review your list here before the weekend.  On the other hand, this change is low-stakes; if we missed one, or had an extra one we shouldn't, we can just update the list.  I don't think changing that even after the PEP is accepted is a problem.  So I'd be satisfied merging your list regardless.